tests/integration/kubernetes/k8s-debug-console.bats

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bats
+# Copyright (c) 2025 Advanced Micro Devices, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+load "${BATS_TEST_DIRNAME}/lib.sh"
+
+# For kata-runtime
+export KATA_HOME="/opt/kata"
+export KATA_RUNTIME="${KATA_HOME}/bin/kata-runtime"
+
+timeout=${timeout:-30}
+
+setup() {
+  if [[ "${K8S_TEST_HOST_TYPE}" != "baremetal" ]]; then
+    skip "Debug console only available from baremetal systems"
```

Comment:
Nit: console is available to all, it's just the test not implemented for all scenarios.

Reply:
Corrected.